src/components/movie_details/MovieDetails.js

```diff
@@ -2,16 +2,39 @@ import React from 'react';
 import './MovieDetails.css';
 
 const MovieDetails = ({ movie, clearSelected }) => {
+  const genres = movie.genres.map(genre => <dd key={movie.genres.indexOf(genre)} className='genre'>{genre}</dd>);
+  const tagline = !movie.tagline ? null : <dd className='tagline'>"{movie.tagline}"</dd>;
+  const budget = !movie.budget ? null : (
+    <>
+      <dt>Budget</dt>
+      <dd>${movie.budget}</dd>
+    </>
+  );
+  const revenue = !movie.revenue ? null : (
+    <>
+      <dt>Revenue</dt>
+      <dd>${movie.revenue}</dd>
+    </>
+  );
+
   return (
     <section className='details-container'>
       <img className='detail-background' src={movie['backdrop_path']} alt=''/>
       <div className='detail-info'>
         <dl>
           <h2 className='detail-title'>{movie.title}</h2>
+          {tagline}
```

Comment:
what happens if tagline/genres/budget/revenue is null? Does it render anything? Does it render "null"?

Reply:
If it's null, it renders nothing! Super handy

Reply:
Great!